package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
We can also add the info section here.
We're missing `name`, `description`, and `version`.

https://docs.npmjs.com/cli/v11/configuring-npm/package-json